app/src/main/java/org/mozilla/fenix/compose/SelectableChip.kt

```diff
@@ -82,17 +88,17 @@ private fun SelectableChipDarkThemePreview() {
 }
 
 @Composable
-@Preview(uiMode = UI_MODE_NIGHT_NO)
-private fun SelectableChipLightThemePreview() {
+@Preview()
```

Comment:
Should this just also have the 2 uiMode?

Reply:
This one overrides the dark/light color tokens, so the second preview isn't necessary here

Reply:
Okay, I am unfortunately imagining a PR where we go through every @Preview and introduce both uiMode and kinda just standardize our @Preview. It will of course not make sense for certain previews, but we should probably retroactively inspect the @Preview we do have and before we have the ability to see the preview both light and dark mode.

Reply:
Gotcha, yeah, I could see us implementing like a `@LightDarkPreview()` annotation to accomplish this. I'm happy to add in the second `@Preview` here then

Reply:
+1 to the `LightDarkPreview` idea